### PR TITLE
Start SPI Fly (OSGi service loader) early

### PR DESCRIPTION
**Pull request by florian-h05**
Related to https://github.com/openhab/openhab-core/issues/4466.

### CODE REVIEW

File: bom/runtime/pom.xml [hunk 1]
```diff
@@ -775,6 +775,18 @@
       <version>${jetty.version}</version>
       <scope>compile</scope>
     </dependency>
+    <dependency>
```
Comment (florian-h05) [inline bom/runtime/pom.xml:778]: FYI I am not sure whether we need those dependencies, but I would think so.

**Comment by J-N-K** [comment]
Please resolve the itests and convert to a PR.

**Comment by florian-h05** [comment]
My distro build works fine without adding `jetty-alpn-server` and `jetty-alpn-java-server` compile dependencies.
Do we need them or not? I cannot really find any information only, but it seems we do not need them as JDK 21 provides ALPN APIs.

**Comment by J-N-K** [comment]
I don't think we need them as `compile` dependencies, it's enough that they are available at runtime.

**Review by J-N-K — APPROVED** [review]

**Pull request merged**